dss/api/bundles/checkout.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+import boto3
+import os
+import requests
+from flask import jsonify
+from cloud_blobstore import BlobNotFoundError
+
+from dss.util.bundles import get_bundle
+from dss.util.checkout import get_execution_id
+from ... import dss_handler, get_logger
+
+
+@dss_handler
+def post(uuid: str, json_request_body: dict, replica: str, version: str = None):
+
+    sfn = boto3.client('stepfunctions')
+    sts_client = boto3.client("sts")
+
+    email = json_request_body['email']
+
+    if replica is None:
+        replica = "aws"
+
+    try:
+        get_bundle(uuid, replica, version)
+
+        sfn_input = {"bundle": uuid, "version": version, "email": email, "replica": replica}
+        if "destination" in json_request_body:
+            sfn_input["bucket"] = json_request_body["destination"]
+        input = json.dumps(sfn_input)
```

Comment:
Please do not overload builtin names. Use a different, more specific name for this variable.

Reply:
renamed to execution_input